### DISCUSSION TITLE
Support for syslog rfc5424 ?

### DISCUSSION THREAD

**Discussion by keuss** [logfellow/logstash-logback-encoder#909] [Q&A] [answered]
Hi,

Does the lib support the RFC 5424 (https://blog.datalust.co/seq-input-syslog/#rfc5424) for syslog or the 3164 RFC ?

Thanks !

**Comment by philsttr** [comment, marked as answer]
No

**Reply by keuss** [reply]
Do you mean "no" for both RFC 5424  AND RFC 3164  ?

- https://blog.datalust.co/seq-input-syslog/#rfc3164
- https://blog.datalust.co/seq-input-syslog/#rfc5424

Thanks

**Reply by philsttr** [reply]
Yes
